.github/workflows/ocserv-version-update.yml

```diff
@@ -0,0 +1,33 @@
+name: bump ocserv version
+on:
+  schedule:
+  - cron: "0 0 * * *"
+jobs:
+  ocserv-version-update:
+    runs-on: ubuntu-latest
+    permissions:
```

Comment:
Do we need this here?

Reply:
Yep - this is a result without the permission section - https://github.com/framebassman/docker-ocserv/actions/runs/7519894802/job/20468951183